tendermint/src/vote.rs

```diff
@@ -92,6 +121,11 @@ impl Type {
     pub fn to_u8(self) -> u8 {
         self as u8
     }
+
+    /// Serialize this type as a 32-bit unsigned integer
+    pub fn to_u32(self) -> u32 {
```

Comment:
Added this because clippy didn't like `to_u8() as u32`

Reply:
Thanks!